src/pages/monetization/app-review-process/index.mdx

```diff
@@ -99,15 +123,15 @@ To make the copy review process as seamless as possible, we have prepared the <a
 
 <SectionLink><a href="https://developers.livechat.com/livechat-app-review-copy-guidelines.pdf" target="_blank" rel="noopener noreferrer">Read the guidelines</a></SectionLink>
 
-### Screenshots
+#### Screenshots
```

Comment:
How about we link our Figma templates here? https://www.figma.com/community/file/1192489623743762740